mdoc/Mono.Documentation/MDocUpdater.cs

```diff
@@ -1102,6 +1158,17 @@ private void CleanupFiles (string dest, HashSet<string> goodfiles)
                             try { System.IO.File.Delete (newname); } catch (Exception) { Warning ("Unable to delete existing file: {0}", newname); }
                             try { typefile.MoveTo (newname); } catch (Exception) { Warning ("Unable to rename to: {0}", newname); }
                             Console.WriteLine ("Class no longer present; file renamed: " + Path.Combine (nsdir.Name, typefile.Name));
+
+                            var framework = frameworks.Frameworks.FirstOrDefault ();
+                            var frameworkXml = oldFrameworkXmls?.FirstOrDefault
+                                (i => i.XPathSelectElements($"Framework/Namespace/Type[@Name='{typeFullName}']").Any());
+                            var frameworkNameAttribute = frameworkXml?.Root?.Attribute ("Name");
+                            if (frameworkNameAttribute != null)
+                            {
+                                var name = frameworkNameAttribute.Value;
+                                framework = frameworks.Frameworks.FirstOrDefault (i => i.Name == name);
+                            }
+                            statisticsCollector.Increment (framework?.Name ?? "", StatisticsItem.Types, StatisticsMetrics.Removed);
```

Comment:
This line has me a bit worried ... there's a lot of uncertainty going on here. What happens if the first parameter is `""`? is an error thrown? Is it covered by a unit test?

This is another case where a code comment would help future maintainers ascertain what's going on.